src/containers/NewUser.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { addUser } from '../actions/index';
+import '../css/newUser.css';
+
+class NewUser extends Component {
+  constructor(props) {
+      super(props);
+      this.state = {
+        name: '',
+        surname: '',
+        sex: 'male'
+      };
+
+      this.handleChangeName = this.handleChangeName.bind(this);
+      this.handleChangeSurname = this.handleChangeSurname.bind(this);
+      this.handleChangeSex = this.handleChangeSex.bind(this);
+      this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChangeName(event) {
```

Comment:
Instead of code duplication, try to use a more generic method for handling input changes, e.g.
`handleInputChange(event, propertyName) =>
this.setState({[propertyName]: event.target.value})`